frontend/server/src/main/java/com/amazonaws/ml/mms/util/CodecUtils/MessageDecoder.java

```diff
@@ -1,18 +0,0 @@
-package com.amazonaws.ml.mms.util.CodecUtils;
```

Comment:
Did you format add moved MessageDecorer and MessageEncoder into to commit?

Reply:
I just refactored the code. This is because the checkstyle doens't like CamelCase for package names.

Reply:
Sorry, I mean for forget to checkin moved class.